.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: Tests using qibolab main
+on: push
+jobs:
+  tests:
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install and configure poetry
+        uses: snok/install-poetry@v1
+        with:
+          virtualenvs-create: true
+          virtualenvs-in-project: true
+      - name: Disable modern installer
+        run: poetry config installer.modern-installation false
+      - name: Install qibolab
+        run: |
+          git clone https://github.com/qiboteam/qibolab
+          cd qibolab
+          poetry install --no-interaction --no-root --with tests --all-extras
+          cd ..
+      - name: Test platforms
+        run: |
+          source $VENV
+          export QIBOLAB_PLATFORMS=$(realpath .)
+          pytest tests/
```

Comment:
```suggestion
          QIBOLAB_PLATFORMS=$(realpath .) poetry run pytest tests/
```

Reply:
This will not work as it is because we need to execute `poetry run` from within the qibolab project directory. Instead I tried 
```sh
cd qibolab
QIBOLAB_PLATFORMS=$(realpath .) poetry run pytest ../tests/
```
 here https://github.com/qiboteam/qibolab_platforms_qrc/actions/runs/6310932947/job/17133888194 but I get that `qibolab` is not installed so maybe the environment is not properly activated.

Reply:
You're right about the change of directory, the problem with installation is not because of environment activation, but because `qibolab` has never been installed before because of the `--no-root` flag

https://python-poetry.org/docs/cli/

Reply:
(`--no-root` means: *install all dependencies but not the package itself*)

Reply:
Thanks, I just blindly copied the other workflow without checking what the options were doing. It is working now.